Makefile

```diff
@@ -173,6 +174,9 @@ help:
 	@echo '                      sourced from ceph-container (e.g., ubuntu:"16.04", centos:"7")'
 	@echo '    e.g., FLAVORS="luminous,ubuntu,16.04 jewel,ubuntu,14.04"'
 	@echo ''
+	@echo '  REGISTRY - The url of the registry to pull images from.'
```

Comment:
s/REGISTRY/BASEOS_REGISTRY/

Reply:
I'm having a thought now that I'm seeing this in context: to keep this related variable consistent with `BASE_IMAGE` below, we should probably either name this `BASE_REGISTRY` OR rename `BASE_IMAGE` to `BASEOS_IMAGE`. That way there isn't a discrepancy between `BASE_` and `BASEOS_` that doesn't exist in the makefile. I think I'm more inclined to rename `BASEOS_REGISTRY` to `BASE_REGISTRY`, but I'm fine either way.